visual-recognition/v3.ts

```diff
@@ -482,7 +482,7 @@ class VisualRecognitionV3 extends BaseService {
    *
    * @param {Object} params - The parameters to send to the service.
    * @param {string} params.classifier_id - The ID of the classifier.
-   * @param {Map<string, NodeJS.ReadableStream|FileObject|Buffer>} [params.positive_examples] - A dictionary that
+   * @param {Record<string, NodeJS.ReadableStream|NodeJS.ReadStream|FileObject|Buffer>} [params.positive_examples] - A dictionary that
```

Comment:
I also set this to `ReadStream` since `createReadStream` returns one. `ReadStream` seems to inherit from `ReadableStream` and worked in my test code:

```ts
  const updateParams: VisualRecognitionV3.UpdateClassifierParams = {
    classifier_id: IBM_CLASSIFIER_ID,
    positive_examples: {
      'my-example': createReadStream(pathToImagesZip)
    }
  }
```

Reply:
I'm curious about this change - would you point me to the documentation you read on this?

Does `ReadableStream` _not_ work for what `createReadStream` returns? Maybe we want to just use `ReadStream` instead, I'm not sure we need to have both